src/components/Authenticate.tsx

```diff
@@ -0,0 +1,28 @@
+import MainPage from './MainPage';
+
+import { Connect } from '@stacks/connect-react';
+import { useAppSelector } from '../redux/store';
+
+const Authenticate = () => {
+  const userSession = useAppSelector((state) => state.userState.userSession);
+  return (
+    <Connect
+      authOptions={{
+        appDetails: {
+          name: 'Stacks React Template',
+          // todo:
```

Comment:
@BowTiedDeployer is this a todo? Then we can create a ticket.

Reply:
is was from the template, it displays the name of the app and its icon when authenticating.
we'll need an icon to be displayed and the name can be `decentralized mining pool`.